test/controller.bar.tests.js

```diff
@@ -251,10 +251,10 @@ describe('Bar controller tests', function() {
 
 		var meta0 = chart.getDatasetMeta(0);
 
-		[	{ b: 290, w: 91, x:  95, y: 161 },
-			{ b: 290, w: 91, x: 209, y: 419 },
-			{ b: 290, w: 91, x: 322, y: 161 },
-			{ b: 290, w: 91, x: 436, y: 419 }
+		[{b: 290, w: 91, x: 95, y: 161},
```

Comment:
That's the only formatting change I don't really like, I prefer the old way because all entries were aligned with a tab. Maybe we should consider to move the first value on a new line:

```javascript
[
    {b: 290, w: 91, x: 95, y: 161},
    {b: 290, w: 91, x: 209, y: 419},
    // ...
].forEach(...
```